### DISCUSSION TITLE
Gérez les perturbations de service sur Kubernetes

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#194] [Announcements]
# Gérez les perturbations de service sur Kubernetes

L'article explore l'outil PDB (Pod Disruption Budget) dans Kubernetes, qui garantit une haute disponibilité des applications lors de maintenances, et essentiel pour éviter les interruptions volontaires.

https://zerofiltre.tech/articles/281

**Comment by imphilippesimo** [comment]
>Notre charge de travail sera donc un déploiement composé de 4 replicas de pods avec comme image nginx. Nous avons également spécifié à l'intérieur de notre fichier de déploiement un podAffinity qui va nous disperser les pods sur tous les nœuds en ordonnant au scheduler de ne placer un pod à côté de l'autre que si les noeuds vides ne le permettent pas.  

Pourrais-tu afficher le code de ce déploiement ?

**Comment by imphilippesimo** [comment]
>Comme d'habitude, vous pouvez retrouver le code complet de cet article ici

Peux-tu ajouter le code complet de ton article en suivant [cette procédure](https://github.com/Zerofiltre-Courses/articles/blob/main/README.md) ?
